contracts/src/v0.8/functions/dev/1_0_0/libraries/FunctionsRequest.sol

```diff
@@ -22,13 +22,14 @@ library FunctionsRequest {
   }
 
   struct Request {
-    Location codeLocation; // The location of the source code that will be executed on each node in the DON
-    Location secretsLocation; // The location of secrets that will be passed into the source code. *Only Remote secrets are supported
-    CodeLanguage language; // The coding language that the source code is written in
-    string source; // Raw source code for Request.codeLocation of Location.Inline, URL for Request.codeLocation of Location.Remote, or slot decimal number for Request.codeLocation of Location.DONHosted
-    bytes encryptedSecretsReference; // Encrypted URLs for Request.secretsLocation of Location.Remote (use addSecretsReference()), or CBOR encoded slotid+version for Request.secretsLocation of Location.DONHosted (use addDONHostedSecrets())
-    string[] args; // String arguments that will be passed into the source code
-    bytes[] bytesArgs; // Bytes arguments that will be passed into the source code
```

Comment:
Why are we losing explainer comments?

Reply:
Must have been lost during merge.  I will add them back.